testapp/home/models.py

```diff
@@ -33,6 +37,11 @@ class HomePage(Page):
         StreamFieldPanel('body'),
     ]
 
+    settings_panels = Page.settings_panels + ReviewDate.settings_panels + [
```

Comment:
Don't need to add an empty array to the end of this line.
Could just be `settings_panels = Page.settings_panels + ReviewDate.settings_panels`